packages/core/src/lib/persisted.ts

```diff
@@ -0,0 +1,88 @@
+import { writable, type Writable } from 'svelte/store';
+
+const stores: Record<string, Writable<unknown> | undefined> = {};
```

Comment:
A module level singleton isn't going to be workable for the SvelteKit app. Why is this record of all stores needed? I'm comfortable leaving it to the caller to avoid key collisions

Reply:
Oh yeah you're right about the window guard. That downgrades my concern to general "be careful about singletons." I think it risks not scaling in the future.

If we're relying on "this singleton is only used in the browser," I think we should make that more obvious in the code / comments to avoid someone shifting it around in the future

Reply:
Yep, agreed. Added a comment for now

Reply:
Are we worried about stores staying in scope even when they're not being used?

Reply:
Hmm good question, I don't think so unless we're using a crazy amount of local / session storage once and then never again in any given client app lifecycle?